editor/scheme_documentation.md

```diff
@@ -530,160 +571,208 @@ environment.
 
 <a class='builtin-header' id='newline'>**`newline`**</a>
 
-    (newline)
+```scheme
+(newline)
+```
 
 Prints a new line.
 
 <a class='builtin-header' id='print'>**`print`**</a>
 
-    (print <val>)
+```scheme
+(print <val>...)
+```
 
-Prints the Scheme representation of `val`. Unlike `display`, this will include
-the outer quotes on a Scheme string and will print a new line.
+Prints the Scheme representation of each `val`, separated by spaces. Unlike `display`, this will include
+the outer quotes on a Scheme string, and a newline.
```

Comment:
"new line" I think, but no biggie.

Reply:
This is copied from the scheme-spec in berkeley-cs61a, I'll change it there first.

Reply:
See pr/4200.